business_requirement_deliverable_cost/models/business.py

```diff
@@ -49,7 +49,8 @@ def _get_partner(self):
         br_id = br_deliverable = False
         if self.business_requirement_deliverable_id.id:
             br_deliverable = self.business_requirement_deliverable_id
-        if br_deliverable.business_requirement_id.id:
+        if self.business_requirement_deliverable_id.\
```

Comment:
Are you getting the same value 3 times or I didn't understand this code?

I mean, What about use?
```python
     @api.multi
     def _get_partner(self):
         self.ensure_one()
         return self.business_requirement_deliverable_id.business_requirement_id.partner_id
```